tests/playwright/src/specs/container-smoke.spec.ts

```diff
@@ -154,7 +154,61 @@ describe('Verification of container creation workflow', async () => {
     await playExpect(startButton).toBeVisible();
   });
 
-  test('Deleting a container', async () => {
+  test(`Start a container from the Containers list`, async () => {
+    const navigationBar = new NavigationBar(page);
+    const containers = await navigationBar.openContainers();
+    const containersDetails = await containers.openContainersDetails(containerToRun);
+    await playExpect(containersDetails.heading).toBeVisible();
+    await playExpect(containersDetails.heading).toContainText(containerToRun);
+    // test state of container in summary tab
+    await pdRunner.screenshot('containers-container-details-start-container.png');
```

Comment:
playwright tracing takes screenshots for everything, there is no need to manually call screenshot here or in any other place.